src/sonic-yang-models/yang-templates/sonic-types.yang.j2

```diff
@@ -95,6 +95,7 @@ module sonic-types {
             enum MIRRORV6;
             enum MIRROR_DSCP;
             enum CTRLPLANE;
+            enum INNERSRCMACREWRITE;
```

Comment:
Why call this as a specific table INNERSRCMACREWRITE? Why not simply L3INNER and can be used for future inner match cases? @bingwang-ms what do you think?